bin/generators/gateway/templates/basic/package.json

```diff
@@ -1,5 +1,8 @@
 {
-  "name": "",
+  "name": "express-gateway-instance",
```

Comment:
Doesn't the CLI overwrite this with the gateway name?

Reply:
If so, then it's overwritten in the wrong moment, and npm is throwing a warning. I need to verify this.

Reply:
It's supposed to happen here: https://github.com/ExpressGateway/express-gateway/blob/master/bin/generators/gateway/create.js#L121

Reply:
The code seems correct, the change is happening *before* the install process.

I can dig into more tomorrow.